### DISCUSSION TITLE
Overleaf中使用的宋体为华文宋体，与Word中不同的问题

### DISCUSSION THREAD

**Discussion by Bread-King** [sjtug/SJTUThesis#800] [提问]
字体如图所示，请问这个有什么方法解决呢？
![QQ截图20220503021512](https://user-images.githubusercontent.com/65778786/166302824-5cf6838d-aba2-46b8-89dc-d4fec79d94cd.png)

**Comment by Bread-King** [comment]
尝试过将字体包上传后用如下代码进行替换，但替换后整个文档编译都会报很多错
`\setCJKfamilyfont{zhsong}{fonts/simsum.ttf}`
`\setCJKfamilyfont{zhhei}{fonts/simhei.ttf}`
`\renewcommand*{\songti}{\CJKfamily{zhsong}}`
`\renewcommand*{\heiti}{\CJKfamily{zhhei}}`

**Reply by LogCreative** [reply]
或许可以先试试 [Wiki 中的方法](https://github.com/sjtug/SJTUThesis/wiki/%E5%9C%A8%E7%BA%BF%E4%BD%BF%E7%94%A8%E8%AF%B4%E6%98%8E#%E5%9C%A8-overleaf-%E4%B8%8A%E9%85%8D%E7%BD%AE%E8%87%AA%E5%AE%9A%E4%B9%89%E4%B8%AD%E6%96%87%E5%AD%97%E4%BD%93)，应当是有效的
